changelogs/fragments/6334-dig-support-multiple-domains.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
```

This is effectively a new feature.
```suggestion
minor_changes:
```

---

Arguably it would be a bug fix since the documentation *claimed* it worked. No?

---

I'm fine with either option, just need someone with authority to make a decision :-)

---

Not really. The implementation has been the way it is for a long time (https://github.com/ansible/ansible/commit/74079db8d218ec6aba8ffe6e6b5dd8596a183d28) *before* the documentation was added (https://github.com/ansible/ansible/commit/24d4787b2d3bf531c048fe3cdce0534180193475#diff-01efc3a6184311a81a00d5bf1cfcab20c63d499cf124dd50d66a1c59ee99bec8R31), and the documentation was wrong since the beginning. So this is a bug in the documentation, not a bug in the plugin, and a bugfix would be to fix the documentation to correctly refer to what the plugin is doing, and not to change the plugin to do what the documentation has been incorrectly claiming.

---

Works for me; will add this change and force-push a branch to be current with `main`.